libr/bin/p/bin_xnu_kernelcache.c

```diff
@@ -723,6 +723,11 @@ static RList *entries(RBinFile *bf) {
 	}
 
 	RKernelCacheObj *kobj = (RKernelCacheObj*) obj->bin_obj;
+	ut64 entry_vaddr = kobj->mach0->entry;
+	ut64 entry_paddr = entry_vaddr - kobj->pa2va_exec;
```

Comment:
can we assume entry_vaddr will be always be bigger than pa2va_exec ? can we have  a check?